ideas/99-confidence.md

```diff
@@ -0,0 +1,121 @@
+
+5. Messages should always be delivered, 100% of the time.
+
+    a. Similar to 1 but stronger
+    b. What happens if mail server goes down?
+    c. Can we implement read/re-send semantics at Whisper level (mailserver)?
+    d. Look into how read receipts can be improved
+    e. Clarify UI a la Whatsapp double-tick
+    f. Consider tracking send/delivered ratio as a metric
+
+6. Offline inboxing not working in public chat.
+
+    a. Clarify perf dependency on smooth-ui swarm
+    b. While perf is being worked on, educate user about offline only being 1-1
+
+7. Perception of lack of notifications.
+    a. UXR: understand expectation -  for non-contacts? Public chats? in-app?
+    b. Separate PN swarm, clarify dependency with this swarm
+
+8. Get notification but nothing visible in app.
+    a. PN delivery being decoupled from Whisper delivery, see point 7 above
+    b. Clarify dependency on PN v2 swarm and where work done
+
+9. Perception of never received messages from other people.
+    a. Investigate if same as filter while offline bug and current limitations
+    b. More 7: Investigate non-contact PN possibility
+    c. More 7: Public chat PN?
+    d. UXR: Understand this problem and perception better
+
+10. Sometimes need to change mail server manually to get messages working.
+    a. Make failure of individual mail server apparent and prompt switching
+    b. Automated switching?
+    c. How ensure no switching is necessary? (HA, see 4)
+
+### Requirements & Dependencies
+
+Request messages history after background: https://github.com/status-im/status-react/pull/3493
+New protocol https://github.com/status-im/status-react/pull/3273
+https://github.com/status-im/ideas/issues/87
+
+### Minimum Viable Product
+<!-- Mandatory, completes the Idea in the fastest route possible, can be hacky, needed to feel progress. See https://imgur.com/a/HVlw3 -->
+Goal Date: <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+Description: <!-- Description of Deliverables-->
+
+**TBD**
+Probably something like: Select subset of above to implement/finish; do initial community check, do design basics and plan initial UXR session.
+
+## Dates
+Goal Date: <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+Description: <!-- Description of Deliverables-->
+
+Testing Days required: <!-- Days required at the end of development for testing -->
+
+## Success Metrics
+**TBD**
+
+95% of a group of 20-100 users surveyed - who don't have additional context beyond Status providing a p2p IM capability - using the app for an extended period of time, should answer 'yes' to the question: "Do you trust Status to deliver messages for you?" (and possibly variants of this).
+
+This is fundamentally a soft or qualitative goal. It is thus necessary but not necessarily sufficient, and additional harder numbers might be used as we develop the capability to measure this.
+
+Zero instabug reports within 30 days of alpha release
+
```

Comment:
Need some form of exit criteria. Not sure what's best here. Thoughts @rasom and @chadyj?